src/Mod/Sketcher/Gui/Workbench.cpp

```diff
@@ -494,7 +494,7 @@ template<>
 inline void SketcherAddWorkbenchTools<Gui::ToolBarItem>(Gui::ToolBarItem& consaccel)
 {
     consaccel  //<< "Sketcher_SelectElementsWithDoFs" //rarely used, it is usually accessed by
-               //solver
+               // solver
                // message.

```

Comment:
Should line 498 be consolidated on to 497 ?

Reply:
This was done by the pre-commit hook to avoid to exceed the 100 columns. But I don't know why three lines of comments are created instead of only 2.

Reply:
weird, then the pre-commit hook is malfunctioning in some way.